### PR TITLE
Updating the readme

### PR DESCRIPTION
**Pull request by Atomme1**
I'm proposing to change the JSON file section to match the real JSON file format you have to implement when doing an recognition training.
Because i have encountered some problem with it ^^' 
Cheers 🥂

### CODE REVIEW

File: references/recognition/README.md [hunk 1]
```diff
@@ -44,15 +44,16 @@ The JSON files must contain word-labels for each picture as a string.
 The order of entries in the json does not matter.
 
 ```shell
-labels = {
-    'img_1.jpg': 'I',
-    'img_2.jpg': 'am',
-    'img_3.jpg': 'a',
-    'img_4.jpg': 'Jedi',
-    'img_5.jpg': '!',
+{
```
Comment (frgfm) [inline references/recognition/README.md:47]: Perhaps we could add:
```shell
# labels.json
```
to make it clear?

Comment (Atomme1) [reply references/recognition/README.md:47]: For the name of the file ? 
Yes if you want but it's already written inside the dataformat sooo as you wish 😄

Comment (felixdittrich92) [reply references/recognition/README.md:47]: @Atomme1 
I think some like this:
To make it really clear :sweat_smile: 
```
# labels.json
{
    "img_1.jpg": "I",
    "img_2.jpg": "am",
    "img_3.jpg": "a",
    "img_4.jpg": "Jedi",
    "img_5.jpg": "!",
    ...
}
```

Comment (frgfm) [reply references/recognition/README.md:47]: Would you mind editing the PR then? :)

File: references/recognition/README.md [hunk 2]
```diff
@@ -43,16 +43,18 @@ Each of these paths must lead to a 2-elements folder:
 The JSON files must contain word-labels for each picture as a string.
 The order of entries in the json does not matter.
 
+# labels.json
```
Comment (frgfm) [inline references/recognition/README.md:46]: That should be put after the "shell" line :)

**Review by frgfm — CHANGES_REQUESTED** [review]
Small mistake of positioning the # labels.json :)

File: references/recognition/README.md [hunk 3]
```diff
@@ -54,6 +53,7 @@ The order of entries in the json does not matter.
     ...
 }
 ```
+# labels.json
```
Comment (frgfm) [inline references/recognition/README.md:56]: I meant right after the line of the shell 🙃  like at the top within the code formatted section, above the opening bracket

**Review by frgfm — CHANGES_REQUESTED** [review]
My bad, I wasn't very clear 😅

File: references/recognition/README.md [hunk 4]
```diff
@@ -43,17 +43,20 @@ Each of these paths must lead to a 2-elements folder:
...
+    "img_1.jpg": "I",
+    "img_2.jpg": "am",
+    "img_3.jpg": "a",
+    "img_4.jpg": "Jedi",
+    "img_5.jpg": "!",
     ...
 }
 ```
 
+When typing your labels, be aware that the VOCAB doesn't handle spaces.
```
[the remark this answers is not in the document]
Comment (felixdittrich92) [reply references/recognition/README.md:58]: @Atomme1 @frgfm I agree, that would be good 👍 Maybe update the text detection README (in this PR) with a short note also ?

[the remark this answers is not in the document]
Comment (frgfm) [reply references/recognition/README.md:58]: Let's do this in another PR :+1:  unless @Atomme1 is willing to update this one

**Review by frgfm — APPROVED** [review]
Thanks @Atomme1 :pray: 

Let's open another PR for the mention of word-level crops

**Comment by frgfm** [comment]
Would you mind merging the "main" branch into your branch @Atomme1 please? :pray:

**Comment by Atomme1** [comment]
> Would you mind merging the "main" branch into your branch @Atomme1 please? 🙏

Heyy, sorry for the late response, i just merge the main into mybranch. But i don't have access to merge my branch on yours

**Comment by Atomme1** [comment]
> > > Would you mind merging the "main" branch into your branch @Atomme1 please? pray
> > 
> > 
> > Heyy, sorry for the late response, i just merge the main into mybranch. But i don't have access to merge my branch on yours
> 
> Sorry @Atomme1, I think you need to merge the main branch again to fix the CI :)

Done ^^

**Comment by frgfm** [comment]
@Atomme1 I think either you forgot to push the merge, or forgot to update the upstream first :) 
I just checked your branch and this line https://github.com/Atomme1/doctr/blob/patch-2/tests/conftest.py#L67
should look like this https://github.com/mindee/doctr/blob/main/tests/conftest.py#L82

**Comment by frgfm** [comment]
To do so:
```shell
git checkout main
git fetch upstream
git merge upstream/main
git push origin main
git checkout patch-2
git merge main
```

then you can push :)

**Comment by Atomme1** [comment]
> 

Okay i think i did it ? 
(i needed to reopen the project on my pycharm xD )

**Comment by felixdittrich92** [comment]
@Atomme1 Thanks a lot for applying all these updates 🤗

**Comment by Atomme1** [comment]
> @Atomme1 Thanks a lot for applying all these updates 🤗

Lessgooo it worked ! 
A pleasure to help the team 😄

**Pull request merged**